src/rpc/rawtransaction.cpp

```diff
@@ -302,7 +303,7 @@ static UniValue gettransactionsarelocked(const JSONRPCRequest& request)
     const NodeContext& node = EnsureAnyNodeContext(request.context);
     ChainstateManager& chainman = EnsureChainman(node);
 
-    CBlockIndex* blockindex = nullptr;
+    const CBlockIndex* blockindex = nullptr;
```

Comment:
Nit: looks like this variable is not used anymore

Reply:
correct, will fix